### ISSUE TITLE
Training CSIKit data

### ISSUE THREAD

**Issue by bachejames** [Gi-z/CSIKit#17]
Hey there!

Thanks for the **incredible** toolkit! I am looking to take the CSIKit data and train it using TensorFlow. The ability to do this has been mentioned, but would love to better understand how to implement this and so have a couple of questions please. Although I imagine this should be tool-agnostic, I am using the ESP32. 

1. How should I append the training data labels to the data I pass into CSIKit? My data looks like [this](https://drive.google.com/file/d/1MUuTdWIGMB03Vv1avKK6PBdga0bKS4A2/view) (data example courtesy of @Gi-z). Could I just create a new column with the label or would CSIKit not accept this?
2. In what format should I export the CSI data from CSIKit?
3. Does anyone have any tutorials they can recommend for training CSI data for novice ML engineers please? Page 5 of [this](https://journals.sagepub.com/doi/pdf/10.1177/1550147718806719) research is the best I have found so far.

Thank you so much!

**Comment by bachejames** [comment]
Closing as the crux of this issue has been resolved. FYI this was that I couldn't read ESP32 data into CSIKit, however this seems to have been caused by a one-off bug in the data from CSIKit (the header row had somehow been mixed up with the first row of data, causing an error).
